ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4620,6 +4620,19 @@ AHU_BSPC_DX5SC_ECONM_EDM_EFSS_EFVSC_SFSS_SFVSC_SSPC_SARC:
   - SSPC
   - SARC
 
+AHU_SFSS_SFVSC_CHWRC_SSPC_FDPSM_SF4FM::
```

Comment:
Please clarify - is the set of fans referred to by `SF4FM` separate from the supply fan referred to by `SFSS` and `SFVSC`? Can you explain a bit more about its control?